file-formats/smtg/type/zif_aff_smtg_v1.intf.abap

```diff
@@ -0,0 +1,47 @@
+INTERFACE zif_aff_smtg_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">General Information</p>
+    "! Combines template header and administrative data.
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">Template Description</p>
+      "! Long description to that email template
+      long_description            TYPE c LENGTH 255,
```

Comment:
We usually recommend having title and name of the component in sync. Is it possible to rename to 
```suggestion
      template_description            TYPE c LENGTH 255,
```
? The other discrepancies between title and name are fine for me.

Reply:
I have updated the description name to "template_description"

Reply:
I waited for some more infos before uploading the latest version. I have no changed the layout of the main view so that two fields will no longer be visible (they are placed on the action popup instead). Will this require another UI review?
![image](https://github.com/user-attachments/assets/e16f39c9-852e-41dd-a560-5cd46403b411)
![image](https://github.com/user-attachments/assets/1641fa13-058a-4993-b4b9-f687d8aeb774)